server.js

```diff
@@ -82,7 +82,21 @@ if (config.SENTRY.DSN) {
           ]
         : [],
     // Half of all requests will be used for performance sampling
-    tracesSampleRate: config.SENTRY.ENVIRONMENT === 'production' ? 0.5 : 0,
+    tracesSampler: samplingContext => {
+      const transactionName =
+        samplingContext &&
+        samplingContext.transactionContext &&
+        samplingContext.transactionContext.name
+
+      if (config.SENTRY.ENVIRONMENT !== 'production') {
+        return 0
+      } else if (transactionName && transactionName.includes('ping')) {
+        return 0
+      } else {
```

Comment:
We can get rid of the `else {}` :)

Reply:
It can, I'm sure this is not my last foray in this piece of code, so I'll clean it up next time :)